### PR TITLE
Refine build.ps1

### PR DESCRIPTION
**Pull request by qinezh**

### CODE REVIEW

**Comment by 928PJY** [comment]
In line50
```
    Write-Error $msg + ", exit code: $lastexitcode"
```

image

should change to this one?
```
   Write-Error "$msg, exit code: $lastexitcode"
```

File: build.ps1 [hunk 1]
```diff
@@ -47,7 +47,7 @@ function ProcessLastExitCode {
     param($exitCode, $msg)
     if ($lastexitcode -ne 0)
```
Comment (hellosnow) [inline build.ps1]: >lastexitcode [](start = 9, length = 12)

should it be $exitCode?

File: build.ps1 [hunk 2]
```diff
@@ -45,9 +45,9 @@ function DotnetPack {
 
 function ProcessLastExitCode {
     param($exitCode, $msg)
-    if ($lastexitcode -ne 0)
+    if ($exitCode.Equals(0))
```
Comment (928PJY) [inline build.ps1]: Question: why `$lastexitcode` before,but '$exitCode' now? should we follow the camel naming rule?

Comment (qinezh) [inline build.ps1]: $lastexitcode is built-in variables in powershell

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**